core/src/lightclient/mod.rs

```diff
@@ -31,14 +60,25 @@ impl From<LightClientError> for jsonrpsee::core::Error {
     }
 }
 
-impl StarknetClient {
+impl StarknetClientImpl {
     pub fn new(starknet_rpc: &str) -> Result<Self> {
         let url = Url::parse(starknet_rpc)?;
         Ok(Self {
             client: JsonRpcClient::new(HttpTransport::new(url)),
+            // kakarot_contract_address: FieldElement::from_hex_be(
+            //     "0x031ddf73d0285cc2f08bd4a2c93229f595f2f6e64b25846fc0957a2faa7ef7bb",
+            // )
+            // .unwrap(),
+            kakarot_account_registry: FieldElement::from_hex_be(
+                "0x052a419fd88f53f9a29d22c3d8db24dd9a9a01a41a483ac660d88622f83c40db",
```

Comment:
let's also put this long value as a top-level const:)

Reply:
```rust
impl StarknetClient {
    pub fn new(starknet_rpc: &str) -> Result<Self> {
        let url = Url::parse(starknet_rpc)?;
        let kakarot_account_registry = FieldElement::from_hex_be(
            ACCOUNT_REGISTRY_ADDRESS,
        )?;
        Ok(Self {
            client: JsonRpcClient::new(HttpTransport::new(url)),
            kakarot_account_registry,
        })
    }
```

Reply:
Ok!